libexec/tgenv-upgrade

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+##########################################
+#
+# This script just works like a software
+# version updater. It's a shortcut to
+# $ git checkout master && git pull
+#
+##########################################
+
+[ -n "${TGENV_DEBUG}" ] && set -x
+source "${TGENV_ROOT}/libexec/helpers"
+
+TGFD=$HOME/.tgenv
+TGGB=$(git branch)
```

Comment:
for readability I strongly recommend avoiding acronyms and abbreviations.

suggest ``current_branch`` as alternative. Google has a pretty extensive style guide for shell scripting which is a good guideline to follow https://google.github.io/styleguide/shellguide.html#s7.2-variable-names

I'm also not sure if this does what you expect?

The output of git branch on my machine currently looks as follows.

```bash
$ git branch
  automatic-detection-on-use
* feature/tgenv-upgrade
  github-actions
  github-actions-2-electric-boogaloo
  main
  tgenv-auto-install-default
  ```

The value of ``$TGGB`` is

```bash
 $ echo "${TGGB}"
  automatic-detection-on-use
* feature/tgenv-upgrade
  github-actions
  github-actions-2-electric-boogaloo
  main
  tgenv-auto-install-default
```

This wont work when we get to Line 21